### PR TITLE
Security: Disable version disclosure to unauthenticated users

### LINKED ISSUE #12925: Add option to disable showing Grafana version in footer & api
The default login page sent by the web server disclosed which version is in use. This information about the exact version of the web
application could aid an attacker to craft targeted attacks against this service.

I could not find a configuration option in the documentation or source code to hide the version information.

### What Grafana version are you using?
v5.2.2
### What datasource are you using?
n/a
### What OS are you running grafana on?
n/a
### What did you do?
HTTP GET /login
### What was the expected result?
No version information disclosure to unauthenticated user.
### What happened instead?
The footer discloses the installed version.
### Include raw network request & response: get by opening Chrome Dev Tools (F12, Ctrl+Shift+I on windows, Cmd+Opt+I on Mac), go the network tab.
`              <li>
                <a href="https://grafana.com" target="_blank">Grafana</a>
                <span>v5.2.2 (commit: aeaf7b2)</span>
              </li>`

### PR DESCRIPTION
**Pull request by frits-v**
New config option to disable the disclosure of version information to users that have not logged in

fixes #12925

### CODE REVIEW

**Review by bergquist — CHANGES_REQUESTED** [review]
Thank you for contributing to Grafana. 

I'm rejecting this until we know how we want to solve this problem.

**Comment by frots** [comment]
Hi @bergquist, that sounds sensible and is actually what the commit does. You'd only need to get rid of the configuration option, but the rest is already there.

File: pkg/api/dtos/index.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ type IndexViewData struct {
 	NewGrafanaVersionExists bool
 	NewGrafanaVersion       string
 	AppName                 string
+	IsSignedIn              bool
```
Comment (bergquist) [inline pkg/api/dtos/index.go]: This property is never assigned any value. 
You can use `User.IsSignedIn` instead.

**Comment by torkelo** [comment]
Anyone who wants to view version info can just request api/health or view it in the grafanaBootData.

**Comment by pgporada** [comment]
I believe this PR should be re-opened.

**Comment by jprenken** [comment]
+1

**Pull request closed without merging**